packages/react/src/page-alert/PageAlert.stories.tsx

```diff
@@ -7,6 +7,9 @@ import { PageAlertTitle } from './PageAlertTitle';
 const meta: Meta<typeof PageAlert> = {
 	title: 'content/PageAlert',
 	component: PageAlert,
+	args: {
+		onClose: undefined,
```

Comment:
```suggestion
		onDismiss: undefined,
```

Can we rename this to `onDismiss` for consistency please 🙏